.github/dependabot.yml

```diff
@@ -0,0 +1,24 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+# Go modules
+- package-ecosystem: "gomod"
+  directories:
+  - "/"
+  - "api/hardwaremanagement"
+  - "api/inventory"
+  - "api/provisioning"
+  schedule:
+    interval: "daily"
+  open-pull-requests-limit: 5
+  groups:
+    k8s:
+      patterns: [ "k8s.io/*", "sig.k8s.io/*" ]
+      update-types: [ "major", "minor", "patch" ]
+    golang:
+      patterns: [ "golang.org/*" ]
+      update-types: [ "major", "minor", "patch" ]
+  #ignore:
+  #  - dependency-name: "*"
+  #    update-types: ["version-update:semver-patch"]
```

Comment:
We could also add checks for github actions
  - package-ecosystem: github-actions
    directory: /
    schedule:
      interval: daily

Reply:
ignore - actions are not running anymore